packages/sanity/src/desk/panes/document/timeline/__workshop__/DefaultStory.tsx

```diff
@@ -0,0 +1,54 @@
+import React, {useMemo} from 'react'
+import {Box, Card, Inline, Stack, Text} from '@sanity/ui'
+import {DeskToolProvider} from '../../../../DeskToolProvider'
+import type {DocumentPaneNode} from '../../../../types'
+import {DocumentPaneProvider} from '../../DocumentPaneProvider'
+import {TimelineMenu} from '../timelineMenu'
+
+const DOCUMENT_ID = 'test'
+const DOCUMENT_TYPE = 'author'
+
+export default function DefaultStory() {
+  const pane: DocumentPaneNode = useMemo(
```

Comment:
I think `useMemo` is a bit unnecessary here since this is a static object. Could be moved to a `const PANE` outside the component scope instead.